Bump dependencies

ianschmitz opened the pull request:
Bump minor/patch versions of packages in preparation for 3.1 release.

mrmckeb commented:
Looks good, but I noticed the CI was cancelled?

ianschmitz commented:
> Looks good, but I noticed the CI was cancelled?

There's actually an issue it looks like. Two test suites are failing. CLI locks then the build times out after 60 mins of waiting it appears. 😕

mrmckeb commented:
Strange, thanks @ianschmitz. Let's keep chasing it.

bugzpodder commented on packages/react-scripts/package.json:
```diff
@@ -55,31 +55,31 @@
     "sass-loader": "7.1.0",
-    "semver": "6.0.0",
+    "semver": "6.3.0",
     "style-loader": "0.23.1",
```
1.0 came out, it fixes sass sourcemaps.

ianschmitz replied:
Bumped to `1.0.0`, however we only enable source maps for CSS in production, so the source maps won't be used by `style-loader` which is only used in dev.

ianschmitz commented:
@iansu @mrmckeb I tracked down the tests hanging issue. It was caused by 
a newer version of `webpack-dev-server`.

It appears to be having issues connecting to the websocket from jsdom:

> TypeError: Cannot read property '_cookieJar' of null 
> at resolve (node_modules/jsdom/lib/jsdom/living/websockets/WebSocket-impl.js:124:39)
> at WebSocketImpl.openingQueues.set.openingQueues.get.then (node_modules/jsdom/lib/jsdom/living/websockets/WebSocket-impl.js:112:94)

There's a related issue over at Jest: https://github.com/facebook/jest/issues/8613.

It may be connected to our usage of `sockjs-client` in `webpackHotDevClient.js` (just a hunch).

I walked back a few versions to `3.6.0` and it was still exhibiting the issue so i left the version where it was for now.

@iansu I fixed an issue with the `DnyamicImport` test suite. It was failing with a console error but the test was passing. I've fixed that here.

There's still a few warnings when running the test suites but i don't think there's anything we can do in the short term.

iansu commented:
Bumping Lerna is a bit scary. There are always issues when we publish. If that happens I'll just revert it.
